### PR TITLE
Updated method of getting battery stats on mac to work properly.

### LINKED ISSUE #489: battery info does not display correctly on mac
Shell: bash
Operating system: OSX El Capitan
Liquid Prompt version (tag, commit): 9c1c8a3

Displaying any battery information on mac seems to be broken altogether.

### PR DESCRIPTION
**Pull request by dogeared**
Fixes #489

### CODE REVIEW

File: liquidprompt [hunk 1]
```diff
@@ -1299,7 +1299,8 @@ case "$LP_OS" in
     {
         (( LP_ENABLE_BATT )) || return 4
         local percent batt_status
-        eval "$(pmset -g batt | sed -n 's/^ -InternalBattery[^	 ]*[	 ]\([0-9]*[0-9]\)%; \([^;]*\).*$/percent=\1 batt_status='\'\\2\'/p)"
+        read percent batt_status <<< $(echo $(pmset -g batt) | awk  -F'[;\t %]' '{print $7, $10}')
```
Comment (dolmen) [inline liquidprompt]: `awk` is forbidden (too slow).

**Comment by dolmen** [comment]
Could you test the patch proposed in #476?

**Comment by dogeared** [comment]
@dolmen - I will look. Thanks.

**Comment by dogeared** [comment]
@dolmen - #476 works on my machine. I will close this PR. Please merge 476! ;) Thanks to you and @thrushcat

**Pull request closed without merging**